FirebaseVertexAI/Sources/VertexLog.swift

```diff
@@ -0,0 +1,112 @@
+// Copyright 2023 Google LLC
```

Comment:
Did this come from somewhere? Otherwise 2024

Reply:
Fixed to 2024.